src/OpenTelemetry.Instrumentation.StartupHook/StartupHook.cs

```diff
@@ -65,4 +78,46 @@ private static string GetLoaderAssemblyLocation()
             throw;
         }
     }
+
+    private static string GetApplicationName()
+    {
+        var applicationName = string.Empty;
+        try
+        {
+            applicationName = AppDomain.CurrentDomain.FriendlyName;
+        }
+        catch (Exception ex)
+        {
+            StartupHookEventSource.Log.Error($"Error getting AppDomain.CurrentDomain.FriendlyName: {ex}");
+        }
+
+        return applicationName;
+    }
+
+    private static bool IsApplicationInExcludeList(string applicationName)
+    {
+        return GetExcludedApplicationNames().Contains(applicationName);
+    }
+
+    private static IList<string> GetExcludedApplicationNames()
```

Comment:
I see no reason to return an interface. It would also improve performance a little 😄 

```suggestion
    private static List<string> GetExcludedApplicationNames()
```